net/BigBuffers.Tests/BigBuffers.Tests.csproj

```
@@ -15,11 +22,14 @@
     <PackageReference Include="NUnit" Version="3.13.2" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
     <PackageReference Include="coverlet.collector" Version="3.1.0">
+
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="StirlingLabs.BigSpans" Version="21.8.1" />
-    <PackageReference Include="StirlingLabs.BigSpans.NUnit" Version="21.8.1" />
+    <PackageReference Include="coverlet.msbuild" Version="3.1.0" PrivateAssets="all" />
+    <PackageReference Include="ReportGenerator" Version="4.8.12" PrivateAssets="all" />
+    <PackageReference Include="StirlingLabs.BigSpans.NUnit" Version="21.7.0" />
```

```suggestion
    <PackageReference Include="StirlingLabs.BigSpans" Version="21.8.1" />
    <PackageReference Include="StirlingLabs.BigSpans.NUnit" Version="21.8.1" />
```
Old version. More correct to include a dependency directly on `StirlingLabs.BigSpans` too.